### DISCUSSION TITLE
Getting lsp-zero to work with NvChad

### DISCUSSION THREAD

**Discussion by julian-hoch** [VonHeikemen/lsp-zero.nvim#353] [Q&A]
I tried to get lsp-zero to work with NvChad, but I am struggling. There seems to be some conflict with the default configuration - in my attempts at following the Tutorial so far either lsp-zero does not load at all, or it loads with an error message ([lsp-zero] Some language servers have been configured before
lsp-zero could finish its initial setup. Some features may fail.), or it loads but still does not do anything.
In NvChad, there is a plugin enabled by default called 'nvim-lspconfig', not sure what the relationship is and if there is perhaps a conflict, so I tried disabling that, but that did not really help either.
I am quite new to NeoVim so I don't fully understand how all the moving parts work together, so I would appreciate some help.
Is it possible / recommended to even try and get lsp-zero to work with NvChad?
Thanks for any hints

**Comment by VonHeikemen** [comment]
Unless there is a specific feature of lsp-zero you want, I don't think you should bother using it with NvChad.

NvChad already provides an "LSP setup", so there is no need for lsp-zero's basic features.
